starters/deno-oak-denodb/src/cache/cache.ts

```diff
@@ -0,0 +1,63 @@
+import { connect, Redis, SetOpts } from '../../deps.ts';
+import { REDIS_CACHE_HOST, REDIS_CACHE_PORT } from '../config/environment.ts';
+import { logger } from '../util/logger.ts';
+
+type GetFreshValue<T> = {
+	(): Promise<T> | T;
+};
+
+const redisHostname = REDIS_CACHE_HOST || 'localhost';
+const redisPort = REDIS_CACHE_PORT ? parseInt(REDIS_CACHE_PORT) : 6379;
+
+export class Cache {
+	redisClient!: Redis;
```

Comment:
I'd expose the `ping` method in this class as well if there's one in the deno redis implementation, because that's useful for the health endpoint. WDYT?